Classes/Manager/FLEXManager.m

```diff
@@ -288,6 +290,15 @@ - (void)registerGlobalEntryWithName:(NSString *)entryName viewControllerFutureBl
     [self.userGlobalEntries addObject:entry];
 }
 
+- (void)setCustomViewerForContentType:(NSString *)contentType viewControllerFutureBlock:(FLEXCustomContentViewerFuture)viewControllerFutureBlock
+{
+    NSParameterAssert(contentType);
+    NSParameterAssert(viewControllerFutureBlock);
+    NSAssert([NSThread isMainThread], @"This method must be called from the main thread.");
+
+    self.customContentTypeViewers[contentType] = viewControllerFutureBlock;
```

Comment:
Perhaps we should lowercase the content type before storing it, since content types are typically case insensitive? What are your thoughts on this?
```suggestion
    self.customContentTypeViewers[contentType.lowercaseString] = viewControllerFutureBlock;
```